container/push.bzl

```diff
@@ -219,6 +219,9 @@ container_push_ = rule(
 )
 
 # Pushes a container image to a registry.
+# You can override some arguments by including adding arguments when running blaze run.
+# Additional arguments will be sent to the command listed in the _pusher rule above.
+# Most common use is adding --dst=myregistry/mypath:debugtag.
```

Comment:
Are you sure? I don't see `--dst` is a valid `bazel run` argument

Reply:
Need the `--` to pass arguments on to rule: `blaze run //target:image_push -- --dst=myregistry/mypath:debugtag`

Reply:
Can you include this example in the comment too? You can also comment that people can run `bazel run //target:image_push -- -help` to get full list of options

Reply:
Done

Reply:
I am not sure if blaze works with rules_docker, but I guess most people use bazel. Can you change it to "bazel"?

Reply:
d'oh, sorry

Reply:
and thank you for reviewing!